google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/ManagedVmDeploymentConfiguration.java

```diff
@@ -48,8 +50,20 @@ public String toString() {
 
   private String dockerFilePath;
   private String appYamlPath;
+  private boolean manualDeployment;
```

Comment:
I was actually just trying to come up with better naming myself - yea thats much better. Also, I've iterated on the UI again improving some things. Will update screenshots.